zerver/tests/test_message_delete.py

```diff
@@ -21,6 +21,151 @@
 
 
 class DeleteMessageTest(ZulipTestCase):
+    def test_do_delete_messages_with_empty_list(self) -> None:
+        realm = get_realm("zulip")
+        do_delete_messages(realm, [], acting_user=None)
```

Comment:
oh, this test should also do a little bit more than just calling `do_delete_messages`. With an empty list, this is supposed to be a noop so maybe just assert that `Message.objects.count()` is the same before and after the call?